sdks/python/apache_beam/io/range_trackers.py

```diff
@@ -28,6 +28,44 @@
            'OrderedPositionRangeTracker', 'UnsplittableRangeTracker']
 
 
+class OffsetRange(object):
+
+  def __init__(self, start, stop):
+    if stop < start:
+      raise ValueError(
+          'Stop offset must not be smaller than the start offset. '
+          'Received %d and %d respectively.', start, stop)
+    self.start = start
+    self.stop = stop
+
+  def __eq__(self, other):
+    assert isinstance(other, OffsetRange)
```

Comment:
<!--new_thread; commit:64a86ed2bb30b5a9fc7127bc2c86deccb4f7a65a; resolved:0-->
Why not return false in this case?